full_utility_rates/California/Huntington Beach City of - 270/hbc-2017-03-04.owrs

```diff
@@ -0,0 +1,85 @@
+---
+metadata:
+  effective_date: 2016-07-04
+  utility_name: "City of Huntington Beach"
+  bill_frequency: monthly
+  # http://www.huntingtonbeachca.gov/government/departments/finance/municipal_services_payments/index.cfm#water-rates
+  # Only a single list of rates given on website with no indication of customer class. I am assuming that it
+  # applies to RESIDENTIAL_SINGLE, RESIDENTIAL_MULTI, COMMERCIAL, and INDUSTRIAL but unclear. Cannot find any other rates
+  
+rate_structure:
+  RESIDENTIAL_SINGLE:
+    service_charge:
+      depends_on: meter_size
+      values:
+         3/4"       : 11.531
+         1"          : 23.062
+         1_1/2"        : 34.593
+         2"          : 57.655
+         3"          : 126.841
```

Comment:
compound and FM are meter_types

it should be formatted

depends_on:
  - meter_size
  - meter_type
values:
  3/4"|FM               : 11.531
  3/4"|compound  : 11.531